types/parity-pmd/parity-pmd-tests.ts

```diff
@@ -22,6 +22,4 @@ format({});
 
 // Invalid sub type
 // $ExpectError
-format({
-    messageType: 1
-});
+format({ messageType: 1 });
```

Comment:
@Flarna is there any update? This just seems like a formatting change and tests are passing as they were

Reply:
There were passing at the time the last change was done in this module but now they fail (see e.g. https://travis-ci.org/DefinitelyTyped/DefinitelyTyped/builds/428047718?utm_source=github_status&utm_medium=notification).

The cause why it fails now without any change in your type definition is that the typescript version used to test has been upgraded.

Reply:
Oh, didn't notice that, I'll check why exactly does this not pass anymore, but this is completely ok. Good catch :)

Reply:
Old TS versions report the error on the `format` call whereas the latest one reports on the concrete property in the passed argument which doesn't match (`messageType` in this case).
I think TS@next is really nice if you pass objects with a lot properties as you get a direct pointer to the property which doesn't match.

Here in the tests it's annoying; would be good if the test framework would allow to specify a range or the typescript version. But this is out of scope for me :grin: